web/src/client/storage.test.js

```diff
@@ -682,475 +662,480 @@ const contexts = {
       }
     };
   },
-  withSystemDevices: () => {
-    managedObjects["/org/opensuse/Agama/Storage1/system/59"] = {
-      "org.opensuse.Agama.Storage1.Device": {
-        SID: { t: "u", v: 59 },
-        Name: { t: "s", v: "/dev/sda" },
-        Description: { t: "s", v: "" }
+  withSystemDevices: () => [
+    {
+      deviceInfo: {
+        sid: 59,
+        name: "/dev/sda",
+        description: ""
       },
-      "org.opensuse.Agama.Storage1.Drive": {
-        Type: { t: "s", v: "disk" },
-        Vendor: { t: "s", v: "Micron" },
-        Model: { t: "s", v: "Micron 1100 SATA" },
-        Driver: { t: "as", v: ["ahci", "mmcblk"] },
-        Bus: { t: "s", v: "IDE" },
-        BusId: { t: "s", v: "" },
-        Transport: { t: "s", v: "usb" },
-        Info: { t: "a{sv}", v: { DellBOSS: { t: "b", v: false }, SDCard: { t: "b", v: true } } },
+      blockDevice: {
+        active: true,
+        encrypted: false,
+        size: 1024,
+        start: 0,
+        recoverableSize: 0,
+        systems: [],
+        udevIds: ["ata-Micron_1100_SATA_512GB_12563", "scsi-0ATA_Micron_1100_SATA_512GB"],
+        udevPaths: ["pci-0000:00-12", "pci-0000:00-12-ata"]
       },
-      "org.opensuse.Agama.Storage1.Block": {
-        Active: { t: "b", v: true },
-        Encrypted: { t: "b", v: false },
-        Size: { t: "x", v: 1024 },
-        Start: { t: "t", v: 0 },
-        RecoverableSize: { t: "x", v: 0 },
-        Systems: { t: "as", v: [] },
-        UdevIds: { t: "as", v: ["ata-Micron_1100_SATA_512GB_12563", "scsi-0ATA_Micron_1100_SATA_512GB"] },
-        UdevPaths: { t: "as", v: ["pci-0000:00-12", "pci-0000:00-12-ata"] }
+      drive: {
+        type: "disk",
+        vendor: "Micron",
+        model: "Micron 1100 SATA",
+        driver: ["ahci", "mmcblk"],
+        bus: "IDE",
+        busId: "",
+        transport: "usb",
+        info: {
+          dellBOSS: false,
+          sdCard: true
+        }
       },
-      "org.opensuse.Agama.Storage1.PartitionTable": {
-        Type: { t: "s", v: "gpt" },
-        Partitions: {
-          t: "as",
-          v: ["/org/opensuse/Agama/Storage1/system/60", "/org/opensuse/Agama/Storage1/system/61"]
-        },
-        UnusedSlots: { t: "a(tt)", v: [[1234, 256]] }
+      partitionTable: {
+        type: "gpt",
+        partitions: [60, 61],
+        unusedSlots: [{ start: 1234, size: 256 }]
       }
-    };
-    managedObjects["/org/opensuse/Agama/Storage1/system/60"] = {
-      "org.opensuse.Agama.Storage1.Device": {
-        SID: { t: "u", v: 60 },
-        Name: { t: "s", v: "/dev/sda1" },
-        Description: { t: "s", v: "" }
+    },
+    {
+      deviceInfo: {
+        sid: 60,
+        name: "/dev/sda1",
+        description: ""
       },
-      "org.opensuse.Agama.Storage1.Partition": {
-        EFI: { t: "b", v: false }
+      partition: { EFI: false },
```

Comment:
If capitalization is OK, then nothing to say. Otherwise, we might need to adjust it accordingly.